pkg/mixin/provider/install.go

```diff
@@ -15,11 +17,73 @@ import (
 )
 
 func (fs *FileSystem) Install(opts mixin.InstallOptions) (*mixin.Metadata, error) {
+	var err error
+	var metadata *mixin.Metadata
 	if opts.FeedURL != "" {
-		return fs.InstallFromFeedURL(opts)
+		metadata, err = fs.InstallFromFeedURL(opts)
+	} else {
+		metadata, err = fs.InstallFromURL(opts)
 	}
+	if err != nil {
+		return nil, err
+	}
+	fs.saveMixinInfo(opts)
+	return metadata, nil
+}
+
+const MixinCacheJSON string = "cache.json"
+
+func (fs *FileSystem) saveMixinInfo(opts mixin.InstallOptions) error {
+	mixinsDir, err := fs.Config.GetMixinsDir()
+	cacheJSONPath := filepath.Join(mixinsDir, "/", MixinCacheJSON)
+	exists, err := fs.FileSystem.Exists(cacheJSONPath)
```

Comment:
Same on this line, since we only care about if it exists or not, and we aren't looking at `err`, please use `exists, _` so that people reading the code don't wonder about why we aren't checking the error coming back.

Reply:
Updated this as well